flink-shaded-hadoop/pom.xml

```diff
@@ -111,6 +111,7 @@ under the License.
 									<include>io.netty:netty:*</include>
 									<include>org.apache.curator:*</include>
 									<include>org.apache.hadoop:*</include>
+									<include>org.mortbay.jetty:*</include>
```

Comment:
Hadoop classes are being included in the fat jar but are not being hidden. That leads to the error scenario, this PR intends to solve.